themes/docsy/layouts/workshops/single.html

```diff
@@ -58,6 +58,7 @@ <h4 class='text-white mb-4'>{{ .Description }}</h4>
 	{{end}}
 </div>
 <div class='container'>
+	{{ partial "stale-notice" (dict "context" . "contentType" "workshop") }}
```

Comment:
Does this make sense for workshops? After a workshop is released, it is commonly updated and that update is purely done in a separate repo outside of TDC. The TDC page tends to remain static even though the workshop itself has been updated.

Reply:
Makes sense. Maybe we need to pull through a 'lastmod' date for the workshops? I'll take a look.